templates/email_change_verify_body.html

```diff
@@ -1,46 +1,203 @@
+                                                    <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="border-collapse:collapse;border-spacing:0px;">
+                                                        <tbody>
+                                                        <tr>
+                                                            <td style="width:320px;">
+                                                                <img alt height="auto" src="{{.Props.SiteURL}}/static/images/welcome_illustration_new.png" style="border:0;display:block;outline:none;text-decoration:none;height:auto;width:100%;font-size:13px;" width="320">
+                                                            </td>
+                                                        </tr>
+                                                        </tbody>
+                                                    </table>
+                                                </td>
+                                            </tr>
+                                            </tbody>
+                                        </table>
+                                    </div>
                                 </td>
                             </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                    <div style="margin:0px auto;max-width:552px;">
+                        <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="width:100%;">
+                            <tbody>
                             <tr>
-                                {{template "email_footer" . }}
+                                <td style="direction:ltr;font-size:0px;padding:0px;text-align:center;">
+                                    <div class="mj-column-per-100 mj-outlook-group-fix" style="font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;">
+                                        <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="vertical-align:top;" width="100%">
+                                            <tbody>
+                                            <tr>
+                                                <td align="center" class="info" style="font-size:0px;padding:0px;word-break:break-word;">
+                                                    <p style="font-family: 'Open Sans';font-style: normal;font-weight: 600;font-size: 16px; line-height: 24px; margin-top: 56px; margin-bottom: 2px; color: #3F4350;">{{.Props.QuestionTitle}}</p>
+                                                    <p style="font-family: 'Open Sans';font-style: normal;font-weight: 400;font-size: 14px;line-height: 20px;margin-top: 2px; color: #3F4350;">{{.Props.EmailInfo1}}<a href='mailto:{{.Props.SupportEmail}}' style='text-decoration: none; color:#2389D7;'>{{.Props.SupportEmail}}</a></p>
```

Comment:
It looks like these two `p` elements are not taking the expected font family:

<img width="610" alt="Screen Shot 2022-09-28 at 4 50 39 PM" src="https://user-images.githubusercontent.com/79058848/192896497-b08732e0-38a4-431c-b6e8-b23332d7582e.png">

Maybe just need to be set as in other elements of the template: `font-family: Open Sans, sans-serif;`